Meadowlark-js/tests/e2e/functions/BulkCreation.ts

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
```

Comment:
Could we change the "functions" directory name to something like "helpers". It seems like these are all test helpers, and "functions" is pretty generic.